browser/iframe-browser.js

```diff
@@ -60,7 +60,7 @@ define(function (require, exports, module) {
      * In which our iFrame will exists and will be filled
      * with the url that has been passed to this function
      */
-    function _update(url) {
+    function _refresh(url) {
```

Comment:
I wonder if we should actually call this `navigate()` since that's more what it's doing